exec/java-exec/src/main/resources/rest/generic.ftl

```diff
@@ -27,7 +27,9 @@
 
       <link href="/static/css/bootstrap.min.css" rel="stylesheet">
 
-      <script src="/static/js/jquery.min.js"></script>
+      <script type="text/javascript" language="javascript" src="//ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
According to this [reference](https://stackoverflow.com/questions/550038/is-it-valid-to-replace-http-with-in-a-script-src-http), it shouldn't be a problem. I will change it though.